scripts/vsc-venv-wrapper/README.md

```diff
@@ -0,0 +1,95 @@
+# Python Virtual Environment Wrapper Script
+
+`vsc-venv` is a script that encapsulates the creation and management of Python virtual environments. 
+This avoids multiple issues with the default `venv` command on the HPC. 
+
+
+One issue is that a virtual environment created for one cluster might not work on another. 
+Additionally, when activating a virtual environment, 
+the same centrally installed modules used during its creation must also be loaded.
+the `vsc-venv` command manages multiple virtual environments for different clusters in a transparent way, 
+while guaranteeing the same module environment.
+
+## Usage
+
+A virtual environment can be activated by running the following command:
+
+```bash
+source ./vsc-venv.sh --activate --requirements <requirements> [--modules module_file]
+```
+
+Here, `requirements` is the path to a file containing the python dependencies to install in the virtual environment.
+An optional `module_file` can be provided, which lists the modules to load before activating the virtual environment.
+
+Automatically, the modules are loaded and the environment is activated. 
+When running this command for the first time, the dependencies from the requirement file are installed.
+
+Now, the software can be run and use packages installed in the virtual environment, along with centrally installed modules.
+
+You can get insights on the current environment using the following commands:
+```
+python --version    # Python version
+pip list            # List of installed Python packages
+module list         # List of loaded modules
+```
+
+To deactivate the virtual environment, run:
+
+```bash
+source ./vsc-venv.sh --deactivate
+```
+
+## Example
+
+Suppose we are on the login node, and we want to make a virtual environment for doduo, which is loaded by default.
+The following files are present in the current directory:
+
+**modules.txt:**
+```bash
+Python/3.12.3-GCCcore-13.3.0
+SciPy-bundle/2024.05-gfbf-2024a
+```
+
+and **requirements.txt:**
+```
+requests
+```
+
+For more info on the `requirements.txt` file, 
+see the [pip documentation](https://pip.pypa.io/en/stable/reference/requirements-file-format/).
+
+We run the following commands to enter the environment
+
+```bash
+source ./vsc-venv.sh --activate --requirements requirements.txt --modules modules.txt
+```
+
+As this creates the virtual environment for the first time, the `venvs` folder is created in the current directory. 
+Within this folder, a new folder is created for the virtual environment `venv-RHEL8-zen2`.
+
+Now, python 3.12 is loaded and the user can use the `requests` package, along with the packages from the SciPy bundle.
+
+To deactivate the virtual environment, run:
+
+```bash
+source ./vsc-venv.sh --deactivate
+```
+
+### joltik
+
+If we want to create a virtual environment for joltik, we can run the following commands:
```

Comment:
Don't use `joltik` as example (it's old, and a GPU cluster), use `donphan` instead:
```suggestion
If we want to create a virtual environment for another cluster, say donphan, we can use `vsc-venv` in an interactive job:
```